packages/account/src/Sections/Security/ApiToken/api-token.tsx

```diff
@@ -1,6 +1,7 @@
 import { WS } from '@deriv/shared';
+import type { TRootStore } from '@deriv/stores/types';
```

Comment:
We don't have `TRootStore` in `@deriv/stores` 🤔
Should be `TCoreStores`, Please update your branch 🙏🏻